### PR TITLE
feat(SWR): add data source swr organizations

### PR DESCRIPTION
**Pull request by luoping-12345**
**What this PR does / why we need it**:
feat(SWR): add data source swr organizations

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/swr" TESTARGS="-run TestAccDataSourceOrganizations_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/swr -v -run TestAccDataSourceOrganizations_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceOrganizations_basic
=== PAUSE TestAccDataSourceOrganizations_basic
=== CONT  TestAccDataSourceOrganizations_basic
--- PASS: TestAccDataSourceOrganizations_basic (22.45s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/swr       22.490s
```

### CODE REVIEW

File: huaweicloud/services/swr/data_source_huaweicloud_swr_organizations.go [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// API: GET /v2/manage/namespaces
```
Comment (jinyangyang222) [inline huaweicloud/services/swr/data_source_huaweicloud_swr_organizations.go:18]: wrong format

File: docs/data-sources/swr_organizations.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+* `name` - (Optional, String) Specifies the name of the organization.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Data source ID.
+
+* `organizations` - All projects that match the filter parameters.
```
Comment (ShiChangkuo) [inline docs/data-sources/swr_organizations.md]:
```suggestion
* `organizations` - All organizations that match the filter parameters.
```

File: huaweicloud/services/acceptance/swr/data_source_huaweicloud_swr_organizations_test.go [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+func testAccDataSourceOrganizations_basic(rName string) string {
+	return fmt.Sprintf(`
+data "huaweicloud_swr_organizations" "test" {}
+
+locals {
+  name = data.huaweicloud_swr_organizations.test.organizations[0].name
+}
+data "huaweicloud_swr_organizations" "filter_by_name" {
+  name = local.name
```
Comment (ShiChangkuo) [inline huaweicloud/services/acceptance/swr/data_source_huaweicloud_swr_organizations_test.go]:
```suggestion
  name = data.huaweicloud_swr_organizations.test.organizations[0].name
```

**Pull request merged**